frontend/src/modules/navbar/Navbar.tsx

```diff
@@ -60,4 +66,8 @@ const Navbar: FC<Props> = props => {
   );
 };
 
-export default withStyles(navbarStyles)(React.memo(Navbar));
+const mapStateToProps = (state: RootStore): StateProps => ({
+  loggedInUser: state.app.user,
```

Comment:
Was there a reason for not doing useSelector for this rather than mapStateToProps?

Reply:
Was just playing around with re-rendering logic. Moved it back as discussed.